.eslintrc

```
@@ -13,7 +13,8 @@
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:prettier/recommended",
-    "prettier"
+    "prettier",
+    "plugin:cypress/recommended"
```

@edwjchen because of how prettier works to turn off certain rules, it should actually be the last plugin that gets defined, so it should instead be: 
```suggestion
    "plugin:cypress/recommended"
    // Do not add plugins below this line. Prettier documents it must be loaded last.
    "plugin:prettier/recommended",
    "prettier"
```
(unfortunately I can't make the suggestion go across all lines because of a limitation of github).

---

note: the suggestion only documents what should be done and cannot be merged as is.